src/sinks/file/path_tz.rs

```diff
@@ -0,0 +1,62 @@
+use chrono_tz::{ParseError, Tz, UTC};
+use std::{
+    convert::{From, TryFrom},
+    default::Default,
+};
+use vector_config::configurable_component;
+
+/// handle tz offset configuration
+/// Defaults to UTC
```

Comment:
```suggestion
/// Configure timezone offset.
```

The default value is provided by the `Default` implementation, so isn't needed here.

Reply:
I've moved the path_tz.rs file to a generic location.